src/main/java/com/harrison/messaging/RabbitAmqpTutorialsRunner.java

```diff
@@ -0,0 +1,22 @@
+package com.harrison.messaging;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.context.ConfigurableApplicationContext;
+
+public class RabbitAmqpTutorialsRunner implements CommandLineRunner {
+
+    @Value("${tutorial.client.duration:3}")
+    private int duration;
+
+    @Autowired
+    private ConfigurableApplicationContext ctx;
+
+    @Override
+    public void run(String... arg0) throws Exception {
+        System.out.println("Ready ... running for " + duration + "ms");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.38.14.86:9000/sonar/coding_rules#rule_key=squid%3AS106)